### DISCUSSION TITLE
page loading takes long....

### DISCUSSION THREAD

**Discussion by delmo254** [pyscript/pyscript#1879] [Q&A]
Is there a way to make the pages load faster

**Comment by WebReflection** [comment]
... which page? :thinking: if it's about pyodide it does take 1 to X seconds accordingly with also packages you are using. You might also use [a splashscreen](https://pyscript.com/@agiammarchi/pyscript-next-splashscreen-copy/latest) if the page doesn't offer much beside Python code or you can move the script to a Worker to avoid blocking, if blocking is the issue you are facing.
